cz.cvut.edu.comparch.qtrvsim.json

```diff
@@ -0,0 +1,33 @@
+{
+    "app-id" : "cz.cvut.edu.comparch.qtrvsim",
+    "runtime" : "org.kde.Platform",
+    "runtime-version" : "6.2",
+    "sdk" : "org.kde.Sdk",
+    "command" : "qtrvsim_gui",
+    "rename-desktop-file" : "qtrvsim.desktop",
+    "rename-appdata-file" : "qtrvsim.metainfo.xml",
```

Comment:
it's sad that it couldn't be named properly the first time. Using the app-id for the name isn't a Flatpak rule. It's part of the XDG spec, include the .desktop and is largely ignore and rarely enforced.

Also looking at upstream, you'll have a problem as the app-id in the metainfo.xml WILL be incorrect.

And you are missing `content_rating` too.

Reply:
that app is also distributed as a package, where non-domain naming is used, so the metainfo naming was kept as original conventions.

it seems that GNOME Software indentify missing content_rating as no age-inappropriate content, so I'd say it's fine (since it does match).

About app-id you're right. For `<launchable>` it gets replaced by flatpak appid, but for `<id>` it's not, however it does work as expected

Reply:
content_rating is required. Otherwise you will not be able to publish on Flathub.

Reply:
> that app is also distributed as a package, where non-domain naming is used, so the metainfo naming was kept as original conventions.

It's a violation of the XDG spec. Just that "distro pacakges" never care about it.
Plenty of distro packaged apps use the proper convention that follow the spec.
BTW the app-id used in the metainfo is actually INVALID.

> it seems that GNOME Software indentify missing content_rating as no age-inappropriate content, so I'd say it's fine (since it does match).

The build will fail. If I say it's missing it's because I know it will fail . GNOME software is more relaxed in its requirements.

> About app-id you're right. For `<launchable>` it gets replaced by flatpak appid, but for `<id>` it's not, however it does work as expected

The launchable is handled by `rename-desktop-file`. 
BTW about the desktop file name, back to what I was saying:
https://specifications.freedesktop.org/desktop-entry-spec/latest/ar01s02.html

Reply:
does this [1] change look reasonable?

[1] https://github.com/cvut/qtrvsim/pull/42

Reply:
Yeah, looks good at first glance.